src/Lucid/Relations/BelongsTo.js

```diff
@@ -32,7 +32,13 @@ class BelongsTo extends BaseRelation {
    * @return {Array}
    */
   mapValues (modelInstances) {
-    return _.map(modelInstances, (modelInstance) => modelInstance[this.primaryKey])
+    const values = []
```

Comment:
It can be done in one line using the `_.transform` method of lodash. I don't like the idea of intermediate variables

Reply:
What u can do is

```js
_.transfor(modelInstances, (result, modelInstance) => {
   if (modelInstance[this.primaryKey]) {
     result.push(modelInstance[this.primaryKey])
   }
   return result
}, [])
```

Reply:
Thx, my knowledge is small in lodash.
I will try use `_.transform` and send changes